neo.UnitTests/UT_Transaction.cs

```diff
@@ -55,40 +66,202 @@ public void Size_Get()
             uut.Script = TestUtils.GetByteArray(32, 0x42);
             uut.Sender = UInt160.Zero;
             uut.Attributes = new TransactionAttribute[0];
-            uut.Witness = new Witness
+            uut.Witnesses = new[]
             {
-                InvocationScript = new byte[0],
-                VerificationScript = new byte[0]
+                new Witness
+                {
+                    InvocationScript = new byte[0],
+                    VerificationScript = new byte[0]
+                }
             };
 
             uut.Version.Should().Be(0);
             uut.Script.Length.Should().Be(32);
             uut.Script.GetVarSize().Should().Be(33);
-            uut.Size.Should().Be(81);
+            uut.Size.Should().Be(82);
+        }
+
+        [TestMethod]
+        public void FeeIsMultiSigContract()
+        {
+            var store = TestBlockchain.GetStore();
+            var walletA = new NEP6Wallet("unit-test-A.json");
+            var walletB = new NEP6Wallet("unit-test-B.json");
```

Comment:
So these files doesn't exist afterall? Just now I'm getting this... I'll change it to something easier to understand.

Reply:
Some test is taking a lifetime to complete, we need to fix that in the future:
`  √ TestSerializeAndDeserializeConsensusContext [2s 74ms]`